chimedb/__init__.py

```diff
@@ -1 +1,3 @@
+from . import mediawiki
```

Comment:
This can't be here.  If you put anything other than the line below in a `pkgutil` subpackage, it breaks the whole system.

The options really are either make a separate package or else include it in `chimedb.core`.